_layouts/base.html.slim

```diff
@@ -251,6 +234,22 @@ body(class="#{page.url.gsub('/','').split('.').first || 'home'} #{page.body_clas
     - page.extra_javascripts.each do |id|
       = deferred_javascripts id
 
+  - if site.metrics
+    / Adobe DTM staging
+    javascript:
+      _satellite.pageBottom();
+      
+    / All metrics and tracking are run via Google Tag Manager
+    javascript:
+      dataLayer = [{'channel' : 'developer'}];
+      (function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
+              new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
+              j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
+              '//www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
+      })(window,document,'script','dataLayer','GTM-NJWS5L');
+    noscript
+      iframe src="//www.googletagmanager.com/ns.html?id=GTM-NJWS5L" height="0" width="0"
+
```

Comment:
Adobe insist that we put this straight after the body. Unless we have a good reason not to, we should do what they ask.